src/lib/merkle_ledger_tests/test_mask.ml

```diff
@@ -697,6 +697,21 @@ module Make (Test : Test_intf) = struct
                   ~message:"All accounts are accessible from m2"
                   ~expect:(Some a) (Mask.Attached.get m2 loc) )
         | _ -> failwith "unexpected" )
+
+  let%test_unit "setting an account in the parent doesn't remove the masked \
```

Comment:
should the different balances be mentioned?

Reply:
Yeah. I've clarified the string a bit.

Reply:
there's some leftover text in the string